react_main/src/components/Popover.jsx

```diff
@@ -561,8 +561,14 @@ export function parseDeckPopover(deck) {
   }
 
   //Words
-  let words = tempParseProfilesToWords(deck.profiles);
-  result.push(<InfoRow title="Words" content={words} key="words" />);
+  // let words = tempParseProfilesToWords(deck.profiles);
+  result.push(<InfoRow title="Profiles" key="profiles" />);
+  deck.profiles.map((profile) => {
+    let avatar = profile.avatar !== undefined;
```

Comment:
avatar is a boolean, if profile.avatar is false but not undefined, this result might be wrong?